src/test/java/seedu/address/logic/parser/profile/ProfileCommandParserTest.java

```diff
@@ -35,4 +54,55 @@ public void parse_profileOption_failure() {
                 ProfileCommand.OPTION_UNKNOWN + ProfileCommand.VALID_FLAGS);
     }
 
+    @Test
+    public void parse_profileOption_add() {
+        Profile profile = new ProfileBuilder().build();
+        try {
+            AddProfileCommand command =
+                    (AddProfileCommand) parser.parse(ProfileUtil.getAddProfileCommand(profile));
+            assertEquals(new AddProfileCommand(profile), command);
+        } catch (ParseException e) {
+            fail();
+        }
+    }
```

Comment:
Maybe it's better to throw the error in the method signature instead of catching it and calling `fail()`. By throwing the exception, in the method signature the error message is more informative. 
```suggestion
    public void parse_profileOption_add() throws Exception {
        Profile profile = new ProfileBuilder().build();
        AddProfileCommand command =
                (AddProfileCommand) parser.parse(ProfileUtil.getAddProfileCommand(profile));
        assertEquals(new AddProfileCommand(profile), command);
    }

Reply:
Yea I agree, thanks for catching that!